lib/que/job.rb

```diff
@@ -68,6 +70,7 @@ def self.enqueue(
           que_job_id: job.attrs["job_id"],
           args: job.attrs["args"],
           **job.attrs.symbolize_keys.slice(*JOB_OPTIONS),
+          **job.get_custom_log_context,
```

Comment:
I like having the method on the instance because it's more easily unit-testable. Do you agree it make more sense doing it this way if we keep the unit tests?

Reply:
The way I see it, we don't need unit tests for this method because we're already testing the behaviour in other higher-level unit tests. But let's keep this method if you feel otherwise!